conf/application.conf

```diff
@@ -21,6 +21,8 @@ appName = income-tax-interest
 
 # Default http client
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
```

Comment:
You don't strictly need to add this if you aren't updating bootstrap as it isn't required in this version